### PR TITLE
Proof DAP_INTERFACE_SUBROUTINES #574

### PR DESCRIPTION
**Pull request by cvsmith**
Resolves https://github.com/chrislgarry/Apollo-11/issues/574

Changes: 
- `HEREIN IS` -> `HEREIN ARE` ([1406](https://www.ibiblio.org/apollo/ScansForConversion/Luminary099/1406.jpg))
  Screen Shot 2020-07-09 at 6 33 18 AM
- `P. S. WEISSMAN` -> `P S WEISSMAN` ([1407](https://www.ibiblio.org/apollo/ScansForConversion/Luminary099/1407.jpg))
  Screen Shot 2020-07-09 at 6 32 57 AM
- Indent line to match scan whitespace ([1409](https://www.ibiblio.org/apollo/ScansForConversion/Luminary099/1409.jpg))
  Screen Shot 2020-07-09 at 6 38 45 AM

### CODE REVIEW

File: Luminary099/DAP_INTERFACE_SUBROUTINES.agc [hunk 1]
```diff
@@ -156,7 +156,7 @@ ALLCOAST	EXTEND			# SAVE Q FOR RETURN
 		QXCH	RUPTREG2
 # Page 1409
 		TC	STOPRATE	# CLEAR RATE INTERFACE.  RETURN WITH A=0
-		LXCH	EBANK		# AND L=EBANK6.  SAVE CALLER'S EBANK.
+		LXCH	EBANK		#   AND L=EBANK6.  SAVE CALLER'S EBANK.
```
Comment (cvsmith) [inline Luminary099/DAP_INTERFACE_SUBROUTINES.agc:159]: I'm not sure about this one - contribution guide makes it sound like we might want 3 spaces, but the scan pretty clearly has the "E" in and "A" lined up

Comment (wopian) [reply Luminary099/DAP_INTERFACE_SUBROUTINES.agc:159]: Contribution guide is just a generalisation of what the layout is like in most of the situations. It sometimes deviates in alignment like here.

**Comment by wopian** [comment]
Thank you again 😄

**Comment by cvsmith** [comment]
You're welcome! I'm sure I'll get bored eventually but for some reason I'm enjoying this kind of task right now :)

**Pull request merged**